# Individual wheel speed information as reported by vehicle

std_msgs/Header header

float64 front_left_wheel_speed      # Measured front left wheel speed in rad/s as reported
float64 front_right_wheel_speed     # Measured front right wheel speed in rad/s as reported
float64 rear_left_wheel_speed       # Measured rear left wheel speed in rad/s as reported
float64 rear_right_wheel_speed      # Measured rear right wheel speed in rad/s as reported
